webapi/Controllers/SpeechTokenController.cs

```diff
@@ -6,15 +6,13 @@
 using System.Threading.Tasks;
 using CopilotChat.WebApi.Models.Response;
 using CopilotChat.WebApi.Options;
-using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
 namespace CopilotChat.WebApi.Controllers;
 
-[Authorize]
```

Comment:
How is this gated now?  What prevents any random dude from obtaining speech service tokens from us now?

Reply:
All controllers have authorization turned on by default, per the code changes in `ServiceExtensions.cs`